consensus/polybft/polybft.go

```diff
@@ -30,9 +30,10 @@ import (
 )
 
 const (
-	minSyncPeers = 2
-	pbftProto    = "/pbft/0.2"
-	bridgeProto  = "/bridge/0.2"
+	minSyncPeers   = 2
+	pbftProto      = "/pbft/0.2"
+	bridgeProto    = "/bridge/0.2"
+	DefaultChainID = 8844
```

Comment:
the old place for this constant is better because chain ID is not polybft-specific term